idoit-install

```diff
@@ -682,7 +704,7 @@ function configureCentOS8 {
     yum --assumeyes --quiet clean all || abort "Unable to clean yum caches"
     rm -rf /var/cache/yum || abort "Unable to remove orphaned yum caches"
 
-    for appStream in httpd:2.4 mariadb:10.3 php:7.2; do
+    for appStream in httpd:2.4 mariadb:10.3 php:7.4; do
```

Comment:
Is there an AppStream for MariaDB `10.5`?

Reply:
Yeah, the change was done some months ago  :-) 
I will create a new updated PR for all OS and app updates ;-)